Seems unrelated?

> Caused by: org.jboss.shrinkwrap.resolver.api.InvalidConfigurationFileException: Found 175 problems while building POM model from /opt/buildAgent/work/e34a6f994de9f7c6/testsuite/integration/basic/pom.xml
> 1/ [ERROR] Non-resolvable import POM: Failed to resolve POM for org.wildfly.core:wildfly-core-parent:21.1.0.Beta1 due to Could not transfer artifact org.wildfly.core:wildfly-core-parent:pom:21.1.0.Beta1 from/to jboss-public-repository-group (https://repository.jboss.org/nexus/content/groups/public/): Error transferring file: Network is unreachable @ org.wildfly:wildfly-common-ee-dependency-management:29.0.0.Beta1-SNAPSHOT

Pretty sure that ran cleanly locally. I'll rerun it here as soon as I can to verify. May restart that job in the interim?

--- a/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/version/JSFDeploymentProcessorTestCase.java
+++ b/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/version/JSFDeploymentProcessorTestCase.java
@@ -29,7 +29,6 @@ import java.net.URL;
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.PropertyPermission;
-
 import jakarta.faces.context.FacesContext;
 
 import org.apache.http.client.methods.CloseableHttpResponse;
@@ -53,7 +52,6 @@ import org.jboss.as.test.integration.jsf.version.ejb.JSFVersionEJB;
 import org.jboss.as.test.integration.jsf.version.war.JSFMyFaces;
 import org.jboss.as.test.integration.jsf.version.war.JSFVersion;
 import org.jboss.as.test.shared.TestLogHandlerSetupTask;
-import org.jboss.as.test.shared.util.AssumeTestGroupUtil;
 import org.jboss.as.test.shared.util.LoggingUtil;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
@@ -63,11 +61,8 @@ import org.jboss.shrinkwrap.resolver.api.maven.Maven;
 import org.jboss.shrinkwrap.resolver.api.maven.PomEquippedResolveStage;
 import org.junit.Assert;
 import org.junit.Assume;
-import org.junit.BeforeClass;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-
 
 /**
  * Tests different ways to add Jakarta Server Faces implementation in ear files
@@ -77,7 +72,6 @@ import org.junit.runner.RunWith;
 @RunWith(Arquillian.class)
 @ServerSetup({JSFDeploymentProcessorTestCase.TestLogHandlerSetup.class})
 @RunAsClient
-@Ignore("WFLY-16528")
 public class JSFDeploymentProcessorTestCase {
 
     private static final String WEB_BUNDLED_JSF = "bundled-jsf";
@@ -98,12 +92,6 @@ public class JSFDeploymentProcessorTestCase {
     @ArquillianResource
     @OperateOnDeployment(WEB_FACES_CONFIG_XML)
     private URL facesConfigXml;
-
-    @BeforeClass
-    public static void beforeClass() {
-        // https://issues.redhat.com/browse/WFLY-15367
-        AssumeTestGroupUtil.assumeSecurityManagerDisabledOrAssumeJDKVersionBefore(11);
-    }
 
     /**
      * Creates a war with all the libraries needed in the war/lib folder, this sample does not call the

